target/linux/kirkwood/files-5.4/arch/arm/boot/dts/kirkwood-linksys-audi.dts

```diff
@@ -165,22 +165,22 @@
 
 		partition@200000 {
 			label = "kernel1";
-			reg = <0x200000 0x290000>;
+			reg = <0x200000 0x300000>;
 		};
 
 		partition@490000 {
 			label = "rootfs1";
-			reg = <0x490000 0x1170000>;
+			reg = <0x500000 0x1160000>;
```

Comment:
You have to calc hex :-)
Size will be 0x1100000 now.

Reply:
That's right. Too fast thinking.